WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -145,6 +145,29 @@ extension URL {
         let components = pathComponents.filter({ $0 != "/" })
         return components.count == 4 && hasWordPressDotComHostname
     }
+
+    /// In case of an app migration, the UUID of local paths can change.
+    /// This method returns the correct path for a given file URL.
+    ///
+    func fixedMediaLocalURL() -> URL? {
```

Comment:
It looks like the only place we're using this (apart from tests) is in the override for `BasePost.featuredImageURL()` — and the logic is also _very similar_ to `-[Media absoluteURLForLocalPath:  cacheDirectory:]`.

If it's only used in one place, I'd rather keep in there, instead of polluting the namespace of `URL` globally. What'd you think?